IPython/core/magics/code.py

```diff
@@ -500,7 +500,7 @@ def _edit_macro(self,mname,macro):
 
     @skip_doctest
     @line_magic
-    def edit(self, parameter_s='',last_call=['','']):
+    def edit(self, parameter_s='', last_call=None):
```

Comment:
Indeed I missed this one as the default arg is modified by a call to `_find_edit_target`.... Good catch.